src/test/java/subway/line/LineAcceptanceTest.java

```diff
@@ -158,20 +152,204 @@ void updateLine() {
     @Test
     void deleteStation() {
         // given
-        ExtractableResponse<Response> response = lineApiCaller.callApiCreateLines(newBunDangLineParams);
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        ExtractableResponse<Response> response = LineApiCaller.callApiCreateLines(newBunDangLineParams);
         String location = response.header("location");
 
         // when
-        response = lineApiCaller.callApiDeleteLine(location);
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.NO_CONTENT.value());
+        LineApiCaller.callApiDeleteLine(location);
 
         // then
-        response = lineApiCaller.callApiFindLines();
-        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        response = LineApiCaller.callApiFindLines();
         List<LineResponse> actual = response.jsonPath().getList(".", LineResponse.class);
         List<LineResponse> expected = Collections.emptyList();
         assertThat(actual).containsAll(expected);
     }
 
+    /**
+     * GIVEN 지하철 노선을 생성하고
+     * WHEN 노선을 수정하면
+     * THEN 수정된 노선이 조회 된다
+     */
+    @DisplayName("지하철노선의 구간을 수정한다.")
+    @Test
+    void updateSections() {
+        // given
+        ExtractableResponse<Response> response = LineApiCaller.callApiCreateLines(newBunDangLineParams);
+        String location = response.header("location");
+
+        // when
+        Map<String, String> params = new HashMap<>();
+        params.put("upStationId", secondStationId.toString());
+        params.put("downStationId", thirdStationId.toString());
+        params.put("distance", "10");
+        LineApiCaller.callApiUpdateSections(params, location);
+
+        // then
+        response = LineApiCaller.callApiFindLine(location);
+        List<Long> actual = response.jsonPath().getList("stations.id", Long.class);
+        Long[] expected = {firstStationId, secondStationId, thirdStationId};
+        assertThat(actual).containsExactly(expected);
+    }
+
+    /**
+     * GIVEN 지하철 노선을 생성하고
+     * WHEN 새로운 구간의 상행역이 기존의 하행역과 일치 하지 않는다면
+     * THEN BadRequest(400) HTTP STATUS 가 발생한다
+     */
+    @DisplayName("새로운 구간의 상행역이 기존의 하행역과 일치 하지 않는다면 Bad Request가 발생한다")
+    @Test
+    void updateSections2() {
+        // given
+        ExtractableResponse<Response> response = LineApiCaller.callApiCreateLines(newBunDangLineParams);
+        String location = response.header("location");
+
+        // when
+        Map<String, String> params = new HashMap<>();
+        params.put("upStationId", thirdStationId.toString());
+        params.put("downStationId", forthStationId.toString());
+        params.put("distance", "10");
+        response = given().log().all()
+                .body(params)
+                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                .when().post(location + "/sections")
+                .then().log().all()
+                .extract();
```

Comment:
단위 테스트에서 한글 변수명을 사용하신 것처럼 인수 테스트에서도 한글 변수나 한글 메서드를 사용해 보시면 어떨까요?
인수 테스트는 시나리오를 검증하는 테스트이니 시나리오 파악에 도움이 될 수 있을 것 같아요
```suggestion
        // given
        Long 신분당선_ID = 지하철_노선_생성(newBunDangLineParams);

        // when
        Map<String, String> params = new HashMap<>();
        ...
        ExtractableResponse<Response> 구간_추가_응답 = 구간_추가(params);
```